lms/djangoapps/lms_xblock/apps.py

```diff
@@ -0,0 +1,25 @@
+"""
+lms_xblock Application Configuration
```

Comment:
Nit: I understand that these apps already exist. But it's unfortunate there's not a uniform name across LMS/CMS for this xblock stuff.

Reply:
Yeah, I figure that would be a separate refactoring PR.